### ISSUE TITLE
SharedFileRestart not working

### ISSUE THREAD

**Issue by lulco** [tomaj/hermes#47]
filemtime doesn't change in while cycle :)

results are cached:
https://www.php.net/manual/en/function.filemtime.php

**Comment by tomaj** [comment]
This should fix it easily - https://www.php.net/manual/en/function.clearstatcache.php

**Comment by lulco** [comment]
but it clear cache stats for all files. I am not sure if this is something I want to run several times per second :)

**Comment by tomaj** [comment]
Fixed here https://github.com/tomaj/hermes/commit/a3c6a2553345b9d2476208cdef681615dd46b602

There is a possibility to clear cache for the particular file

**Comment by lulco** [comment]
OK, I tried it with first parameter clear_realpath_cache = true and it didn't work for me :) Maybe I misunderstood docs:

```
filename
Clear the realpath and the stat cache for a specific filename only; only used if clear_realpath_cache is true.
```

Can you backport this fix to older versions too? I have installed version 2.2.2

**Comment by tomaj** [comment]
I've released 2.2.3 - check and let me know if it is working as expected.
